packages/eslint-config-ckeditor5/.eslintrc.js

```diff
@@ -483,6 +483,12 @@ module.exports = {
 					{
 						methodNames: METHODS_THAT_USE_AS_CONST_INSTEAD_OF_RETURN_TYPE
 					}
+				],
+				'ckeditor5-rules/ckeditor-plugin-flags': [
```

Comment:
As these rules are used only in CKEditor 5 repos (Core/CC), we should modify their `eslint` configuration directly instead of disallowing these properties in all projects where the rules are used.

Reply:
Please take a look at config of this particular PR: https://github.com/cksource/ckeditor5-commercial/pull/6693/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5